docs/web-configurator.md

```diff
@@ -115,12 +169,32 @@ Buzzer Speaker
 * `Buzzer Pin` - The GPIO pin used for the buzzer.
 * `Buzzer Volume` - Audio volume of buzzer. 0-100.
 
+Extra Button Configuration
+
+![GP2040 Configurator - Extra Button](assets/images/gpc-add-ons-extra-button.png)
+
+* `Extra Button Pin` - The GPIO pin used for the extra instance of a button.
+* `Extra Button` - The button that will have an extra instance.
+
+Player Number (X-INPUT ONLY)
+
+![GP2040 Configurator - Player Number](assets/images/gpc-add-ons-player-number.png)
+
+* `Player Number` - Choose what player number this RP2040 based device will be representing.  This is only for X-INPUT mode.
+
+SOCD Selection Slider
```

Comment:
Looks like some of the SOCD content is in the Data Backup and Restoration section.